system/HTTP/Message.php

```diff
@@ -351,6 +328,21 @@ public function prependHeader(string $name, string $value)
 		return $this;
 	}
 
+	/**
+	 * Takes a header name in any case, and returns the
+	 * normal-case version of the header.
+	 *
+	 * @param string $name
+	 *
+	 * @return string
+	 */
+	protected function getHeaderName(string $name): string
+	{
+		$lower_name = strtolower($name);
```

Comment:
I know you're just moving around this method but can we take this opportunity and change the variable name from `$lower_name` to `$lowerName`?

Also, there is one code divider (//------) left after this method.

Reply:
Removed the divider and the use of the interim variable.

Reply:
it seems we can use is rector rule for that, that can be to update entire codebase

https://github.com/rectorphp/rector/blob/master/rules/coding-style/src/Rector/Variable/UnderscoreToCamelCaseVariableNameRector.php